proto/pdpb.proto

```diff
@@ -73,6 +73,16 @@ service PD {
 
     rpc UpdateServiceGCSafePoint(UpdateServiceGCSafePointRequest) returns (UpdateServiceGCSafePointResponse) {}
 
+    rpc GetAllServiceGroups(GetAllServiceGroupsRequest) returns (GetAllServiceGroupsResponse) {}
```

Comment:
Can we use standalone like `gcpb.proto` ?

Reply:
do you mean both the `message` and `rpc` or just the `message`?
and should we also move old GC interface to a separate file?

Reply:
It's hard to separate out `gc` related messages as they depends on `RequestHeader` and `ResponseHeader`